### ISSUE TITLE
External Memory

### ISSUE THREAD

**Issue by codelauncher444** [princeton-nlp/TRIME#8]
Does this architecture really learn from an external memory? Suggest that I have a pre-trained model like GPT-2 and I have a large text file. Can I use continued pretraining and use just the trime loss and change the training objective so that the model fully captures the knowledge in the text file.  If this idea is feasible, how can I use this codebase to implement?
